src/tests/system/tests/readme.rst

```diff
@@ -0,0 +1,115 @@
+SSSD System Tests
+=================
+
+Tests are written in Python using our `SSSD <https://tests.sssd.io/en/latest/>`__ framework and `pytest <https://docs.pytest.org/en/stable/>`__ framework and most notably `pytest mh <https://github.com/next-actions/pytest-mh>`__  plug-in amongst others. For more information please visit our `SSSD framework wiki <https://tests.sssd.io/en/latest/>`__.
+
+.. note::
+
+    We are in the progress of overhauling our all of our tests. This large housekeeping effort, there is an emphasis on providing excellent test coverage, clear concise documentation and easy to maintain test code. These tests are not just system tests but will contain some integration, but will be marked.
```

Comment:
`our all of our tests`

Reply:
Two `but` on the same (last) sentence.

Reply:
`This large housekeeping effort, there is...`

Shouldn' t it be 'In this large housekeeping...` or `With this large...`?

Reply:
I have reworded the entire thing.

Reply:
Nice! Thanks.

Should we mention how tests will be marked?